### PR TITLE
[test] Check recents and directory have standards

### PR DESCRIPTION
**Pull request by roc**
* will fail if recents on the homepage drop below 3
* will fail if there are not 10 results on the directory homepage

### CODE REVIEW

File: ui/components/Dataset/index.js [hunk 1]
```diff
@@ -19,9 +26,11 @@ function Model({ model }) {
       </Link>
       <p>{description}</p>
       <Flex className="nhsuk-body-s">
-
         <p className={classnames('nhsuk-body-s', styles.noBottom)}>
-          Type: <Tag type={standard_category} classes="nhsuk-body-s">{standard_category}</Tag>
+          Type:{' '}
+          <Tag type={standard_category} classes="nhsuk-body-s">
+            {standard_category}
+          </Tag>
```
Comment (lennym) [inline ui/components/Dataset/index.js:33]: `{'🤷‍♂️'}`

File: ui/test/specs/standards/list.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
   it('should accesss standards listing page', async () => {
     await browser.url(`/standards`);
-
     await expect($('ul')).toExist();
     await expect($('h1=Browse the standards directory')).toExist();
   });
+
+  it('There should be results in the directory ', async () => {
+    await browser.url(`/standards`);
+    await expect($('#browse-results')).toExist();
```
Comment (lennym) [inline ui/test/specs/standards/list.js:10]: 😻

File: ui/test/specs/home.js [hunk 3]
```diff
@@ -14,4 +14,10 @@ describe('Homepage', () => {
       )
     );
   });
+
+  it('should show recent standards on the right hand side', async () => {
+    await browser.url(`/`);
+    const $recents = await $$('ul.nhsuk-u-font-size-16 li');
```
[the remark this answers is not in the document]
Comment (roc) [reply ui/test/specs/home.js:20]: Good shout, I'll do the same as with the directory index test 👍

File: ui/components/Dataset/index.js (hunk 1 above)
Comment (roc) [reply ui/components/Dataset/index.js:33]: Prettier formats it like this, boo! https://github.com/prettier/prettier/issues/4223 
https://github.com/prettier/prettier/issues/4223#issuecomment-1104943499 *sigh*

**Pull request merged**